### PR TITLE
Depend on tfjs1.0 in toxicity.

### PR DESCRIPTION
**Pull request by annxingyuan**
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-models/165)

### CODE REVIEW

File: toxicity/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tensorflow-models/toxicity",
-  "version": "0.1.0",
+  "version": "0.1.1",
```
Comment (dsmilkov) [inline toxicity/package.json]: bump to 1.0.0. Can you also make sure the demo uses the latest. Also run yarn to update the yarn.lock file.

**Comment by annxingyuan** [comment]
@dsmilkov thanks - this should be fixed now?

File: toxicity/demo/package.json [hunk 2]
```diff
@@ -9,8 +9,8 @@
     "node": ">=8.9.0"
   },
   "dependencies": {
-    "@tensorflow-models/toxicity": "0.0.2",
-    "@tensorflow/tfjs": "^1.0.0-alpha3"
+    "@tensorflow-models/toxicity": "0.1.0",
+    "@tensorflow/tfjs": "~1.0"
```
Comment (dsmilkov) [inline toxicity/demo/package.json]: make sure peer deps are with ^

File: toxicity/package.json [hunk 3]
```diff
@@ -13,13 +13,13 @@
     "url": "https://github.com/tensorflow/tfjs-models.git"
   },
   "peerDependencies": {
-    "@tensorflow/tfjs": "^1.0.0-alpha3"
+    "@tensorflow/tfjs": "~1.0"
```
Comment (dsmilkov) [inline toxicity/package.json]: peer deps should start with ^. here and for other models too.

**Comment by annxingyuan** [comment]
FYI Travis is failing because Universal Sentence Encoder 1.0.0 hasn't been published yet so lockfile is invalid, but following the steps we talked about this error should resolve itself - I'll publish the Universal Sentence Encoder first and restart travis.

**Pull request merged**